lib/wordsmith/project.rb

```diff
@@ -1,24 +1,28 @@
-class Wordsmith::Project
-  attr_reader :name, :slug, :schema, :templates
+module Wordsmith
+  class Project
+    attr_reader :name, :slug, :schema, :templates
 
-  def self.all
-    projects_attributes = Wordsmith.client.get 'projects'
-    projects_attributes.map {|p| new(**p)}
-  end
+    def self.all
+      projects_attributes = Wordsmith.client.get('projects')
+      projects_attributes.map {|p| new(**p)}
+    end
 
-  def self.find(slug)
-    project = all.find { |p| p.slug == slug }
-    project or fail %Q(Project not found with slug: "#{slug}")
-  end
+    def self.find(slug)
+      project = all.find { |p| p.slug == slug }
+      project || fail(%Q(Project not found with slug: "#{slug}"))
```

Comment:
It actually needs parentheses here believe it or not! Syntax error surprisingly. I think I'm going to just add them to all fails